test/e2e/rds/resources/db_subnet_group_2az.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBSubnetGroup
+metadata:
+  name: $DB_SUBNET_GROUP_NAME
+spec:
+  name: $DB_SUBNET_GROUP_NAME
+  description: $DB_SUBNET_GROUP_DESC
+  subnetIDs:
+    - $SUBNET_AZ1
+    - $SUBNET_AZ2
+  tags:
+    - key: key1
+      value: value1
```

Comment:
I figure there will be a suite of tests that only test tags, considering the complexity planned for them in the future?

Reply:
yes.